e2e/testnet_wallet.test.js

```diff
@@ -0,0 +1,216 @@
+
+  // calculating total blocks in this sync process
+  const blockstotalNum = blockssyncednowNum + blocksnotyetsyncedNum;
+
+  // getting blocks per batch or batch size from the screen
+  const batchsize = element(by.id('syncreport.blocksperbatch'));
+  const batchsize_attributes = await batchsize.getAttributes();
+  const batchsizeNum = Number(batchsize_attributes.text);
+
+  log.info('batches', batchNum);
+  log.info('total batches', batchesNum);
+  log.info('blocks sync now', blockssyncednowNum);
+  log.info('blocks not yet sync', blocksnotyetsyncedNum);
+  log.info('blocks total', blockstotalNum);
+  log.info('batch size:', batchsizeNum);
+
+  // a couple of examples:
+  // batch: 1  -> means blocks between 0 and 100
+  // batch: 33 -> means blocks between 3200 and 3300
+  if (blockssyncednowNum < (batchNum * batchsizeNum) - batchsizeNum || blockssyncednowNum > (batchNum * batchsizeNum)) {
+    fail('The synced blocks are not align with the synced batches');
+  }
+  
+  if (blockstotalNum < (batchesNum * batchsizeNum) - batchsizeNum || blockstotalNum > (batchesNum * batchsizeNum)) {
+    fail('The total blocks in this process are not align with the total of batches');
+  }
+
+  await element(by.text('CLOSE')).tap;
+}
+// export { sendTestnet };
+
+const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
+
+describe('...', () => {
+  it('loads a testnet wallet', loadTestnet);
+  
+  it('syncs...', syncTestnet);
+
+  it('adds return address to the memo if that option is selected, and correctly renders confirm screen', async () => {
+    await element(by.text('SEND')).tap();
+    
+    await element(by.id('send.addressplaceholder')).replaceText(
+      // 'u1lx7nlnqqqs7p4hcqz4hyjgnw7h8zskcgr2f8dfhm96np0gnfdzu7jtms7q2pxd7ufy96wxzdsdy65jvp3td7fj2ltcz0jpak86ddyszl9ykn5s86q3xataya5867z3tj2x8cw0ljyrenymr2gcqmjf50gmexqj233yn3kdaxn2yukwcx87emurufakf82wapgnu5h3fvae6aw9uus2r',
+      'utest1cvzrmq2qy9sc7mk7ramrc4gr0uu4m0mdc89srrx5k94vs6em3kt5x8zdtu8k8w6v56yfn79yr0frzuyzgl8uztgrwdsfkadkce2num4lhplxp7dgayjdtzhs4mgvf95ntt6rqj472y430huvhmk4g5nrlq932vswp4xqlxr3j5j4e74667dzzyxpfdrhr8qg235dfg4y8373kqnuz4e',
+    );
+    await element(by.id('send.amount')).replaceText('0.01');
+    // await element(by.id('send.checkboxUA')).tap();
+    await element(by.id('send.scrollView')).scrollTo('bottom');
+    await element(by.id('send.memo-field')).replaceText("8008");
+    await element(by.id('send.scrollView')).scrollTo('bottom');
+    await element(by.id('send.button')).tap();
+
+    await element(by.id('send.confirm.scrollView')).scrollTo('bottom');
+    
+    const memo = element(by.id('send.confirm-memo'));
+
+    await expect(memo).toBeVisible(100);
+    await expect(memo).toHaveText(
+      '1\n2\n3\n4\n5\n6\n7\n8\nReply to: \nutest1kcth7yst8s7u4yqf9hnrrfmrte40sy49z83qctvmxc528xf37swhkw53yhhzmceq07a83waevggwh3uyxffjgx30wcua2s0rgpt3seeae4nlrdedj9kxdgcx9uyug3d0ae5rk4q22r8pl37h75pnfaqvrp3qqxvnvvfldvs0frhllv7av8lhcesaskmsx7gfw5dd8shhm9aju98aelv',
```

Comment:
Here you are expecting 8 lines with the numbers... but you put in this field the value `8008`... I don't understand it, maybe I'm missing something.